pkg/cmd/cli/backup/describe.go

```diff
@@ -85,6 +86,7 @@ func NewDescribeCommand(f client.Factory, use string) *cobra.Command {
 
 	c.Flags().StringVarP(&listOptions.LabelSelector, "selector", "l", listOptions.LabelSelector, "only show items matching this label selector")
 	c.Flags().BoolVar(&details, "details", details, "display additional detail in the command output")
+	c.Flags().BoolVar(&insecureSkipVerify, "insecureskipverify", insecureSkipVerify, "accept any TLS certificate presented by the storage service")
```

Comment:
Would it be good to add something like `not secure`, or `not recommended for production` to all the cli documentation?

Reply:
`do not verify the TLS certificate for storage requests. This is susceptible to man-in-the-middle attacks.` -- think `man-in-the-middle attacks` says it's dangerous. since this is the client and the flag is non-permanent, I kept it shorter.